src/rabbit_auth_backend_ldap.erl

```diff
@@ -314,14 +323,16 @@ with_login(Creds, Servers, Opts, Fun) ->
                 {UserDN, Password} ->
                     case eldap:simple_bind(LDAP, UserDN, Password) of
                         ok ->
-                            ?L1("bind succeeded: ~s", [UserDN]),
+                            ?L1("bind succeeded: ~s",
```

Comment:
This log entry will be scubbed with `network`, but not with `true` log verbosity. This doesn't make mush sense to me.